wallet/src/account/mod.rs

```diff
@@ -774,6 +778,17 @@ impl Account {
         self.account_info.update_best_block(best_block_height, best_block_id);
         db_tx.set_account(&self.key_chain.get_account_id(), &self.account_info)?;
 
+        Ok(new_tx_was_added)
+    }
+
+    pub fn sync_best_block(
```

Comment:
Please document this function.

I have a feeling this function name is not really indicative to what this function is doing... after you document it let's talk again about it.